.github/workflows/build.yml

```diff
@@ -96,6 +111,23 @@ jobs:
           # Only need to retain for a day since the beta builder slurps it up to distribute.
           retention-days: 1
 
+      - name: Update reminder with testing instructions
+        id: update-reminder-comment
+        uses: actions/github-script@v6
+        if: ${{ github.event_name == 'pull_request' && steps.set-reminder-comment.outputs.result != null }}
+        env:
+          BRANCH_NAME: ${{ github.head_ref }}
+        with:
+          script: |
+            const { BRANCH_NAME } = process.env;
+            const commentBody = `You can now test your Pull Request on WordPress.com. On your sandbox, run \`bin/jetpack-downloader test jetpack ${ BRANCH_NAME }\` to get started.`;
+            await github.rest.issues.updateComment( {
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              body: commentBody,
+              comment_id: ${{steps.set-reminder-comment.outputs.result}},
```

Comment:
This should probably be passed through via an environment variable, like you're doing with `BRANCH_NAME`. Even though it should always be an integer, it's probably better to follow good patterns.

Reply:
Done in bf586a55f799967187718129d1ab9e112dc65370